libkbfs/disk_block_cache.go

```diff
@@ -851,6 +922,55 @@ func (cache *DiskBlockCacheLocal) evictLocked(ctx context.Context,
 	return cache.evictSomeBlocks(ctx, numBlocks, blockIDs)
 }
 
+// GetLastUnrefRev implements the DiskBlockCache interface for
+// DiskBlockCacheLocal.
+func (cache *DiskBlockCacheLocal) GetLastUnrefRev(
+	ctx context.Context, tlfID tlf.ID) (kbfsmd.Revision, error) {
+	cache.lock.Lock()
+	defer cache.lock.Unlock()
+	err := cache.checkCacheLocked("Block(GetLastUnrefRev)")
+	if err != nil {
+		return kbfsmd.RevisionUninitialized, err
+	}
+
+	rev, ok := cache.tlfLastUnrefs[tlfID]
+	if !ok {
+		// No known unref'd revision.
+		return kbfsmd.RevisionUninitialized, nil
+	}
+	return rev, nil
```

Comment:
Did you purposely not check the underlying disk cache since it's synced to memory?

Reply:
Yeah exactly.  No reason to go to disk.